frontend/src/container/ReduxTlmDispatcher.jsx

```diff
@@ -88,10 +89,12 @@ export class ReduxTlmDispatcher extends React.Component {
 
   handleContentModified = data => {
     this.props.dispatch(updateWorkspaceContentList([data.content], data.workspace.workspace_id))
+    this.props.dispatch(updateSearchResultList(data.content))
```

Comment:
couldn't the search result reducer be updated by the action `updateWorkspaceContentList` ?

That would mean the `reducer/searchResult.js` would handle the action `${UPDATE}/${WORKSPACE_CONTENT}`

Reply:
Functionally, there are no restrictions. The only thing "wrong" would be the naming, because the search result has no connection with the workspace.

Reply:
that's true.
But I don't think it is an issue.
The word "dispatch" in `props.dispatch()` is meaningful, it broadcast the action and every reducers that are interested in implements it

I think it's better than having multiple actions for the same thing

Reply:
Ok, works for me. I'm gonna change. :smile: